mmv1/third_party/terraform/services/resourcemanager/resource_google_project_iam_member_remove.go

```diff
@@ -0,0 +1,132 @@
+package resourcemanager
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-google/google/tpgiamresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+	cloudresourcemanager "google.golang.org/api/cloudresourcemanager/v1"
+)
+
+func ResourceGoogleProjectIamMemberRemove() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGoogleProjectIamMemberRemoveCreate,
+		Read:   resourceGoogleProjectIamMemberRemoveRead,
+		Delete: resourceGoogleProjectIamMemberRemoveDelete,
+
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Required:    true,
+				Description: `The project id of the target project.`,
+			},
+			"role": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Required:    true,
+				Description: `The target role that should be removed.`,
+			},
+			"member": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Required:    true,
+				Description: `The IAM principal that should not have the target role.`,
+			},
+		},
+		UseJSONNumber: true,
+	}
+}
+
+func resourceGoogleProjectIamMemberRemoveCreate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Do we expect any race conditions when creating this within a large Terraform config? For example, depending on the ordering of creates in a large config a binding may be applied before or after the member remove resource.

I guess there's not much we can do there and the recreate should appear on next plan time if it is applied in the wrong order

Reply:
Yep, exactly. I'll add some more details to the docs.